src/app/components/download-listings/download-listings.jsx

```diff
@@ -79,6 +80,7 @@ const allCategories = [
   { name: 'Practice Type', key: 'practiceTypeName' },
   { name: 'Real World Testing', key: 'rwt' },
   { name: 'SVAP', key: 'svap' },
+  { name: 'Risk Management Summary Information', key: 'riskManagementSummaryInformation' },
```

Comment:
From the "API Documentation" category down, all of these elements are alphabetized. This one should be above SVAP to keep that consistent. I'd also move the headerName from line 64 above the svap columns to match